SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialEventTracer.cpp

```diff
@@ -0,0 +1,278 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Interop/Connection/SpatialEventTracer.h"
+
+#include "Misc/ScopeLock.h"
+#include "SpatialGDKSettings.h"
+#include "UObject/Object.h"
+#include "UObject/UnrealType.h"
+#include <WorkerSDK/improbable/c_io.h>
+#include <WorkerSDK/improbable/c_trace.h>
+
+DEFINE_LOG_CATEGORY(LogSpatialEventTracer);
+
+using namespace SpatialGDK;
+using namespace worker::c;
+
+SpatialEventTracerGuard EventTracerGuard;
```

Comment:
This should be in an anon namespace. But I'm confused as to why the mutex isn't part of the class instead of being here?